src/main/java/seedu/address/model/ReadOnlyBookShelf.java

```diff
@@ -23,6 +23,8 @@
      */
     ObservableList<Book> getBookList();
 
+    public String getKey();
```

Comment:
remove.

Reply:
I need this to get Data.key

Reply:
The key shouldn't be in the book shelf data.